scripts/vcs_wrapper

```diff
@@ -15,6 +15,11 @@ for file in $files; do
   file_args+=( "$file" )
 done
 
+filelists="$WAKE_IP_RESOURCE_FILELIST"
+filelistargs=( )
+for filelist in $filelists; do
+ filelistargs+=(-f "$filelist")
```

Comment:
Thanks for adding this. Could I make one small request here that instead of using the `-f` flag, we use the `-F` flag? `-F` is similar to `-f` except that it interprets path inside the filelist as relative to the path of the filelist. E.g. `vcs -F /path/to/filelist.F` will prepend `/path/to` to each path inside of `filelist.F`. This allows us to more easily change the paths to the files in the future if we preserve the relative path between the `.F` file and the actual Verilog files.

Reply:
Agree, will change to `-F`